### ISSUE TITLE
Auto-refresh PDF file

### ISSUE THREAD

**Issue by maoshunyu** [nvarner/typst-lsp#76]
- [x] I have searched the [issues](https://github.com/nvarner/typst-lsp/issues) of this repo and believe that this is not a duplicate.
- [x] I have searched the [discussions](https://github.com/nvarner/typst-lsp/discussions) and believe that my question is not already covered.

## Feature Request

Pull request #72 seems to create or open the generated pdf in VSCode only after push the "Show PDF" Button.
Could you please make the pdf file auto refreshed after compiling like the latex/markdown extensions in vscode.

**Comment by dvdvgt** [comment]
You can already specify whether the PDF file is auto-refreshed on save or on change in VSCode's settings for the extension. 
However, not all PDF viewer extensions seem to support reloading the PDF file on changes. I can recommend this [PDF viewer](https://marketplace.visualstudio.com/items?itemName=mathematic.vscode-pdf) for that purpose.

**Comment by Shilionz** [comment]
If you are using Windows, have a try with [SumatraPDF](https://www.sumatrapdfreader.org/free-pdf-reader).  
This supports reloading PDF file on change but it is also a nice FOSS PDF reader.

**Comment by maoshunyu** [comment]
> If you are using Windows, have a try with [SumatraPDF](https://www.sumatrapdfreader.org/free-pdf-reader). This supports reloading PDF file on change but it is also a nice FOSS PDF reader.

It seems that although Export PDF is onChange, Sumatra will only refresh when i save .typ file

**Comment by Enter-tainer** [comment]
You can take a look at https://github.com/Enter-tainer/typst-preview-vscode, which supports markdown preview-like behavior by writing changes to a shadow file. Note that it's still in its early stage.

**Comment by Shilionz** [comment]
> 

(With most recent Typst 0.4.0) Configure the setting "Export PDF" for Typst LSP to `onType`. 
SumatraPDF works well with this setting and it can update file really fast in my machine.
